### PR TITLE
fix(new doc for aerospike): created new doc

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    src={infrastructureAerospikeDashboard}
+/>
+
+<figcaption>
+After setting up our Aerospike integration, we give you a dashboard for your Aerospike platform metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx:26]: This time, this is not an optional step.
```
### Step 1: Install the infrastructure agent [#infra-install]
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx:28]: Installing the infrastructure agent will get you important metrics about the host where your app is running.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent) for the host containing your Aerospike app.
+
+### Step 2: Install nri-aerospike application [#aerospike]
+
+First, find the prometheus exporter application named `nri-aerospike`. Nri-aerospike is available in the apt-get repository.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx:34]: The prometheus exporter application named `nri-aerospike` is available on the apt-get repository.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        label_source: aerospike
+
+        exporter_files_path: /tmp
+
+        # transformations:
+        #   - description: "General processing rules"
+        #     ignore_metrics:
+        #     - prefixes:
+        #       - "go_"
+        #       - "process_"
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx:74]: lines 69 to 74 can be removed

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+Your Aerospike dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [infrastructure agent](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
+
+Here's an example NRQL query checking the percentage of memory capacity free on the namespace:
+
+```
+FROM Metric 
+SELECT latest (aerospike_namespace_memory_free_pct)
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx:99]: SELECT latest(aerospike_namespace_memory_free_pct)

**Review by pkosanam — CHANGES_REQUESTED** [review]
Thank you for the PR @homelessbirds ,

Kindly use the below dashboard image

dashboard image: https://github.com/newrelic/newrelic-quickstarts/blob/892f8e1395a843137f7a8afc14e71ee48c04aa52/dashboards/aerospike/aerospike01.png

**Review by akristen — DISMISSED** [review]
Looks good! Left some copy edits.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+First, find the prometheus exporter application named `nri-aerospike`. Nri-aerospike is available in the apt-get repository. 
+
+1. To install nri-aerospike, use the command below:
+
+```
+sudo apt-get install nri-aerospike
+```
+You'll know nri-aerospike was successfully installed because:
+*  You will see that the **Exporter** is installed in the `/var/db/newrelic-infra/newrelic-integrations/bin` directory
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx]: fixed

**Pull request merged**